src/devices/src/virtio/block/request.rs

```diff
@@ -123,7 +126,7 @@ impl PendingRequest {
 
         FinishedRequest {
             // Account for the status byte
```

Comment:
I guess this comment should no longer be here

Reply:
You're right. Done.